share/commands/agent-unset-force-audit

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# @description authorize rudder-agent to be run in non audit mode
+# @man Cancel the change made by rudder agent set-force-audit
+# @man +
+# @man *Options*:
+# @man +
+# @man *-q*: run the command in quiet mode (display only error messages)
+# @man +
+# @man *-c*: run the command without color output
+
+. "${BASEDIR}/../lib/common.sh"
+
+QUIET=false
+
+while getopts "qc" opt; do
+  case $opt in
+    q)
+      QUIET=true
+      ;;
+    c)
+      clear_colors
+      ;;
+  esac
+done
+
+rm -f /opt/rudder/etc/force-audit-agent
+
+[ $? -ne 0 ] && printf "${RED}error${NORMAL}: Rudder force audit mode flag could not be removed.\n" && exit 1
```

Comment:
The user does not have to know it is implemented using a flag file. We could only say something like "Rudder force audit mode could not be disabled"